locales-src/newtab.ftl

```diff
@@ -19,3 +19,20 @@ newtab-search-box-search-the-web-text = Search the Web
 newtab-search-box-search-the-web-input =
     .placeholder = Search the Web
     .title = Search the Web
+
+## Top Sites form dialog
+
+newtab_topsites_add_header = New Top Site
+newtab_topsites_edit_header = Edit Top Site
+newtab_topsites_title_label = Title
+newtab_topsites_title_placeholder =
+    .placeholder = Enter a title
+
+newtab_topsites_url_label = URL
+newtab_topsites_url_placeholder =
+    .placeholder = Type or paste a URL
```

Comment:
Not sure if these "-placeholder" parent ids would be better as…

```ftl
newtab-topsites-url-input =
  .placeholder = …
```

(I do see both patterns (and more 😛 ) if you click into various files and look at the parent id https://searchfox.org/mozilla-central/search?q=.placeholder&path=.ftl )

Reply:
Ah.... Definitely sounds redundant now. I'll switch to the input suggestion.